apps/src/gamelab/TextConsole.jsx

```diff
@@ -0,0 +1,169 @@
+/** @file Text display over sprite lab visualization column */
+import PropTypes from 'prop-types';
+import React from 'react';
+import {Transition} from 'react-transition-group';
+
+const lineHeight = 18;
+const maxHeight = lineHeight * 6;
+
+export const styles = {
+  hide: {
+    display: 'none'
+  },
+  console: {
+    display: '',
+    background: 'rgba(200,200,200,0.5)',
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    zIndex: 2,
+    width: '100%',
+    lineHeight: lineHeight,
+    transitionProperty: 'max-height',
+    transitionDuration: '1000ms'
```

Comment:
can this be done with the [CSSTransition](http://reactcommunity.org/react-transition-group/css-transition) Component instead?

Reply:
Looks like CSSTransition also needs this transitionDuration. This may be possible with react-spring, however. I'll spend a bit of time looking into that.

Reply:
After some time looking at react-spring, I'm not going to switch to it for this change. When we get a clearer idea of the UI requirements, I'll revisit this.